tox.ini

```diff
@@ -89,20 +88,19 @@ deps =
     httplib2: httplib2
     urllib3: urllib3<2
     boto3: boto3
-    boto3: urllib3
+    boto3: urllib3<2
```

Comment:
This change seems unrelated. If it is, it should either be a separate pull request or just be dropped please. Thank you!